hack/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "cincinnati-graph-data"
+version = "0.1.0"
+authors = ["Vadim Rutkovsky <vrutkovs@redhat.com>"]
+edition = "2018"
```

Comment:
Is this some Cargo magic, or should it be 2020?

Reply:
Rust has two major editions - [2015 and 2018](https://doc.rust-lang.org/edition-guide/editions/index.html)